opensoundscape/torch/train.py

```diff
@@ -81,9 +80,14 @@ def train(
             targets = y.squeeze(1)
 
             if spec_augment:
+                # X is currently shape [1, 3, width, height]
+                # Take to shape [1, 1, width, height] for use with `augment`
+                X = X[0][0].unsqueeze(0).unsqueeze(0)
```

Comment:
Is this right? I read this as take the first image in the batch only.

Reply:
Seems like `X[:][0].unsqueeze(0)`?

Reply:
Fixed by https://github.com/kitzeslab/opensoundscape/pull/179/commits/1ea680bd155ff542b622389947df723254c17c98